TestProjects/Playground/Playground.iOS/Setup.cs

```diff
@@ -14,8 +14,8 @@ public Setup(MvxApplicationDelegate applicationDelegate, UIWindow window)
         {
         }
 
-        public Setup(MvxApplicationDelegate applicationDelegate, IMvxIosViewPresenter presenter)
-            : base(applicationDelegate, presenter)
+        public Setup(MvxApplicationDelegate applicationDelegate, UIWindow window, IMvxIosViewPresenter presenter)
```

Comment:
If I'm not missing anything, then this is a big breaking change. The constructor 
`public Setup(MvxApplicationDelegate applicationDelegate, IMvxIosViewPresenter presenter)` doesn't exist anymore on `MvxIosSetup`. Do we need this change?